templates/form/lang_select.html.twig

```diff
@@ -2,12 +2,12 @@
 {%- block choice_widget_options -%}
     {% for group_label, choice in options %}
         {%- if choice is iterable -%}
-            <optgroup label="{{ choice_translation_domain is same as(false) ? group_label : group_label|trans({}, choice_translation_domain) }}">
```

Comment:
I'm not quite understanding what `choice_translation_domain` is meant to represent in this file. I don't think it's being used from the controller, unless I'm looking in the wrong place which is possible. Maybe it was meant to be `choice.translation_domain`, on the iterator? As there are instances of `translation_domain`

Reply:
ah it's coming from https://symfony.com/doc/current/reference/forms/types/language.html#choice-translation-domain which is what I was missing